src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderSingleLine.java

```diff
@@ -0,0 +1 @@
+package com.puppycrawl.tools.checkstyle.checks.imports.customimportorder; import static java.io.File.createTempFile; import java.util.Map; class InputCustomImportOrderSingleLine {}
```

Comment:
Need more tests. An input with empty line in first line and import groups in single line would trigger false positive result, due to the limitation of `CustomImportOrderCheck#getCountOfEmptyLinesBefore(int)`.
Actually we need `getCountOfEmptyLinesBetween(int line1, int line2)` to cover this case.